flocker/acceptance/testtools.py

```diff
@@ -157,6 +151,10 @@ def _clean_node(test_case, node):
     # http://doc-dev.clusterhq.com/advanced/cleanup.html#removing-zfs-volumes
```

Comment:
This line isn't a change in this branch/story. If a link is wrong, please create a separate issue for it.